backend.go

```diff
@@ -144,10 +183,10 @@ func (b *kubeAuthBackend) loadConfig(ctx context.Context, s logical.Storage) (*k
 	if err != nil {
 		return nil, err
 	}
+	// We know the config is empty so exit early
 	if config == nil {
-		return nil, errors.New("could not load backend configuration")
+		return config, nil
```

Comment:
Another place that we're calling `loadConfig` is: 
https://github.com/hashicorp/vault-plugin-auth-kubernetes/blob/9a0d8f05f80dfd6a4053b7a303f1afe23ac11138/path_login.go#L88-L93

We should also nil-check in there to avoid panics. Would it be good to also call out this behavior in `loadConfig`'s doc-string so future users know that they should nil-check the config? Are there any test breakages that would result from this change (I see `loadConfig` being called directly a handful of times in tests)?

Reply:
Updated in [139b7d6](https://github.com/hashicorp/vault-plugin-auth-kubernetes/pull/142/commits/139b7d6100e9856ac19f1f010f6fbd85fb3fe249)